pymongo/topology.py

```diff
@@ -213,7 +213,7 @@ def _select_servers_loop(self, selector, timeout, address):
             # No suitable servers.
             if timeout == 0 or now > end_time:
                 raise ServerSelectionTimeoutError(
-                    self._error_message(selector))
+                    self._error_message(selector)+", Timeout: "+str(timeout)+"s, Topology Description: "+repr(self.description))
```

Comment:
A couple suggestions:

1. We try to keep our lines limited to 80 characters. Do you mind wrapping? (Eventually we'll start using black/pep8 for auto-formatting.)
2. Usually we use format strings instead of using `+`. Do you mind reformatting to `"%s, Timeout: ...." % (self._error_message(selector), timeout, self.description)`?